src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -51,6 +46,13 @@ protected RepositoryApiClientImpl(String baseUrl, HttpRequestHandler httpHandler
                 .config()
                 .setObjectMapper(objectMapper);
 
+        // Add compression header if a client is created
+        if (httpClient != null) {
+            httpClient
+                    .config()
+                    .setDefaultHeader("Accept-Encoding", "gzip");
```

Comment:
If the user calls `getDefaultRequestHeaders`, I don't think this Accept-Encoding header will be returned. I think this should be fixed if you use the `setDefaultRequestHeaders` function to set the default header instead.

Reply:
ok done, and also updated the `setDefaultRequestHeaders()` function so that it won't automatically delete the gzip compression header when adding in another header